src/BloomBrowserUI/bookEdit/js/textOverPicture.ts

```diff
@@ -411,7 +441,52 @@ export class TextOverPictureManager {
                 }
             };
         }
-        // ENHANCE: Have ctrl+click go through the text box (currently text box intercepts the click, which is desirable in many cases)
+        // ENHANCE: Have ctrl+click go through the text box
+    }
+
+    // Recomputes the coordinates of element relative to the specified origin element's info
```

Comment:
That is, elementX and elementY are relative to the top left of elementBounds? or relative to the origin of elementBounds? And we want coordinates relative to the top left (or origin?) of originBounds? But offset by its top and left border? I think a bit more comment here might be helpful.
Is it only because we know element doesn't have a border that we don't need to consider that?